src/pepr/operator/controllers/keycloak/authservice/authservice.spec.ts

```diff
@@ -26,42 +26,41 @@ vi.mock("../../istio/ambient-waypoint", () => ({
   setupAmbientWaypoint: vi.fn().mockResolvedValue(undefined),
 }));
 
-// Mock the K8s API
+// --- Pepr K8s and R utility mock ---
 const mockGet = vi.fn();
 const mockUpdate = vi.fn();
 const mockApply = vi.fn();
 const mockList = vi.fn();
 
-vi.mock("kubernetes-fluent-client", async importOriginal => {
-  const actual = await importOriginal<typeof import("kubernetes-fluent-client")>();
+vi.mock("pepr", async importOriginal => {
+  const actual = (await importOriginal()) as Record<string, unknown>;
```

Comment:
nit: in other places I think we have this typed more specifically:
```suggestion
vi.mock("pepr", async () => {
  const actual = await vi.importActual<typeof import("pepr")>("pepr");
```